packages/preview/humanities-cv/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "humanities-cv"
```

Comment:
This name seems a little too descriptive. We try to avoid too canonical names, so multiple packages serving the same purpose can coexist without one having an unfair advantage. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules)

Reply:
Sure! I've updated the name to "humanistically"-- hope that works.